src/main/java/edu/greenblitz/tobyDetermined/IsRobotReady.java

```diff
@@ -0,0 +1,40 @@
+package edu.greenblitz.tobyDetermined;
+
+
+import edu.greenblitz.tobyDetermined.commands.swerve.MoveToGrid.Grid;
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Elbow;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Extender;
+
+public class IsRobotReady {
+	public static boolean isArmReady(){
+		switch (Grid.getInstance().getSelectedHeight()){
+			case HIGH:
```

Comment:
DOC

Reply:
code is as self explanatory as can be doc would sound the same